integration/up/autocreate_test.go

```diff
@@ -192,7 +204,14 @@ func TestUpAutocreateV2(t *testing.T) {
 	require.NoError(t, commands.RunOktetoCreateNamespace(oktetoPath, namespaceOpts))
 	defer commands.RunOktetoDeleteNamespace(oktetoPath, namespaceOpts)
 	require.NoError(t, commands.RunOktetoKubeconfig(oktetoPath, dir))
-	c, _, err := okteto.NewK8sClientProvider().Provide(kubeconfig.Get([]string{filepath.Join(dir, ".kube", "config")}))
+	config := kubeconfig.Get([]string{filepath.Join(dir, ".kube", "config")})
+	for _, authInfo := range config.AuthInfos {
+		homeEnvVar := api.ExecEnvVar{Name: constants.OktetoHomeEnvVar, Value: dir}
+		authInfo.Exec.Env = append(authInfo.Exec.Env, homeEnvVar)
```

Comment:
in this to add okteto in the path right? should it not be the responsibility of the client instead the test?